src/main/java/com/example/demo/YoutubeApiJson.java

```diff
@@ -0,0 +1,33 @@
+package com.example.demo;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class YoutubeApiJson {
+    public class PageInfo {
+        public int totalResults;
+        public int resultsPerPage;
+    }
+    public String kind;
+    public String etag;
+    public String nextPageToken;
+    public String prevPageToken;
+    public PageInfo pageInfo;
+    @JsonProperty("items")
+    public YoutubeVideoJson[] youtubeVideoJsons;
```

Comment:
直さなくてもいいんですが、
```suggestion
    public YoutubeVideoJson[] items;
```
でもバインドできそうかなと言う共有です。（`@JsonProperty("items")`は不要にできそう

Reply:
Youtube API documentの命名方法を従うから名前を付けました。Jsonの名前と違いますからそう書いています。

Reply:
> Youtube API documentの命名方法

これがどこから来てるは良くわかってないんですが、レスポンスの json プロパティ名と java のフィールド名合わせると何がダメなのかわかってないです。他は合わせてますよね？？

https://developers.google.com/youtube/v3/docs/videos/list?hl=ja
```
{
  "kind": "youtube#videoListResponse",
  "etag": etag,
  "nextPageToken": string,
  "prevPageToken": string,
  "pageInfo": {
    "totalResults": integer,
    "resultsPerPage": integer
  },
  "items": [
    動画リソース
  ]
}
```

Reply:
すみません。あの時何の資料を参考したちょっと覚えてないですから、公式の通り直したいと思います。

Reply:
修正しました。ご確認ください！